src/earl-loader.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+const globby = require('globby')
+
+/**
+ * Load jsonld/earl report(s)
+ * - if there are multiple reports, this function concatanates them into one report
+ *
+ * @method earlLoader
+ * @param {String|String[]} pathOrGlob file path to jsonld/earl report
+ * @returns {Object[]}
+ */
+const earlLoader = async pathOrGlob => {
+	if (!pathOrGlob) {
+		throw new Error(`argument is missing`)
+	}
+
+	const reports = globby.sync(pathOrGlob).map(reportPath => {
+		const fileContent = fs.readFileSync(reportPath, { encoding: 'utf-8' })
+		return JSON.parse(fileContent)
```

Comment:
You may want to try / catch this thing, so you can throw an error that include the filename in the error. Makes debugging this much easier.